titiler/api/endpoints/mosaic.py

```diff
@@ -175,6 +177,9 @@ async def mosaic_tile(
 
     tilesize = 256 * scale
 
+    # TODO: Parametrize/support other tms
+    tms = morecantile.tms.get(TileMatrixSetNames.WebMercatorQuad.name)  # type: ignore
```

Comment:
I'm good supporting only WebMercator, supporting other tms will be hard

Reply:
Why do you say that?  Would it be hard mapping `rio_tiler_crs` across a list of assets compared to `rio_tiler`?

Reply:
I was thinking about the quadkey!

Reply:
I haven't looked at morecantile in a second, is there a feasible way of validating 2x decimation for a particular tile matrix set?  So that those which fit the quadkey tiling scheme may be supported and those that don't would not be supported.

Assuming that a tile matrix set has 2x decimation I think it is realistic to support here, please correct if that is wrong.

Reply:
well in fact I'm not sure if the TMS in the tiler should be == in the tms/quadkey used for the mosaic 🤷‍♂️ maybe it's time to think about another way to index the tiles

Reply:
For now I removed morecantile and am using `mercantile` to calculate tile bounds.